index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tribute Page</title>
+  </head>
+  <body>
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <div id="img-div">
+        <img
+          id="image"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+          alt="image" />
+        <div id="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+          on how to increase wheat yields - part of his life-long war on hunger.
+        </div>
```

Comment:
Please try using HTML semantic tags like `<article></article>` for easier readability to your code and to offer greater accessibility.

Reply:
- [ ] [OPTIONAL] Kindly use emojis and gifs in your comments to make them human-friendly. 🙂️
- [ ] [OPTIONAL] It would be very nice if you provide some links to a resource about the changes you are requesting.
Example
"You can check this [article](https://www.freecodecamp.org/news/semantic-html5-elements/) from freecodecamp to get more information about semantic HTML."